user_tools/src/spark_rapids_pytools/resources/qualification-conf.yaml

```diff
@@ -61,6 +61,7 @@ toolOutput:
 sparkRapids:
   mvnUrl: 'https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark-tools_2.12'
   repoUrl: '{}/{}/rapids-4-spark-tools_2.12-{}.jar'
+  toolsJarRegex: 'rapids-4-spark-tools_2.12-.*.jar'
```

Comment:
Don't we need that for profiling as well?

Reply:
That is a good catch. Thanks @amahussein